packages/mobile/locales/en-US/sendFlow7.json

```diff
@@ -48,6 +48,7 @@
   "walletAddress": "Wallet Address",
   "inviteWithoutPayment": "Hi {{name}}! I’ve been using {{appName}} to send money worldwide. It’s easy to try and I want to invite you to check it out with this link: {{link}}",
   "inviteWithEscrowedPayment": "Hi {{name}}! I’ve been using {{appName}} to send money worldwide. I just sent you {{amount}} Celo Dollars. Please download the {{appName}} App and redeem your Celo Dollars using this link: {{link}}",
+  "inviteWithEscrowAndNoCode": "Welcome to Valora, {{name}}! I've sent you Celo Dollars. Go to {{link}} to get started",
```

Comment:
Is there a reason to use this and not `inviteWithEscrowedPayment`? The wording on the other string is the one on [this doc](https://docs.google.com/document/d/1DftgFh0PShTWjynMAbGzaoDOknh0MGSexfVCuJyPgc4/edit#heading=h.yrdzg1i0xig3). Also, the `{{name}}` doesn't actually go here, since we're using the name of the user using the app, which doesn't make a lot of sense. It should be removed from the other strings in master.